ADOPTERS.md

```diff
@@ -6,30 +6,35 @@ If you are a Chaos Mesh adopter, and are willing to share your Chaos Mesh story,
 
 - [Apache APISIX](https://github.com/apache/apisix)
   - Apache APISIX is a dynamic, real-time, high-performance open source API gateway, which provides rich traffic management features, such as load balancing, dynamic upstream and more.
-  - APISIX integrates Chaos Mesh into open-source CI pipelines, to further enhance APISIX's resiliency and robustness. See [Use case](https://chaos-mesh.org/blog/How-Chaos-Mesh-Helps-Apache-APISIX-Improve-System-Stability)
+  - APISIX integrates Chaos Mesh into open-source CI pipelines, to further enhance APISIX's resiliency and robustness. See [Use case](https://chaos-mesh.org/blog/How-Chaos-Mesh-Helps-Apache-APISIX-Improve-System-Stability).
 - [ByteDance](https://bytedance.com/en/)
   - ByteDance is a technology company operating a range of content platforms including TikTok, Lark, Helo, Vigo Video, Douyin, and Huoshan in over 150 markets and 75 languages.
   - ByteDance's self-developed chaos engineering platform is mainly used by the company's own technology system. As there are some cloud-native deployment services involved, they integrated Chaos Mesh as the underlying fault injection engine, which is a key supplement to ByteDance’s chaos engineering platform.
 - [Celo](https://celo.org/)
 - [Dailymotion](https://www.dailymotion.com/)
-- [DataStax Fallout](https://github.com/datastax/fallout)
+- [DataStax](https://www.datastax.com/) 
+  - DataStax utilized Chaos Mesh to create a tool for running local or large scale remote based distributed correctness, verification and performance tests: [Fallout](https://github.com/datastax/fallout), which they use to [test AstraDB](https://youtu.be/Kw7gMurHJnQ), a multi-cloud DBaaS built on Apache Cassandra
```

Comment:
Is "utilized" inappropriate? Should we use the past tense here?

Reply:
Well, they **used** Chaos Mesh to create Fallout the tool, and **uses** this tool to test AstraDB, so I reckon we should use past tense here? 🤔